guides/Getting-Started.md

```diff
@@ -6,13 +6,17 @@ We'll start with the installation of the compiler and Spago build tool, and then
 
 #### Installing the Compiler
 
-You'll need [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node) and to be [able to install global packages](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory) to proceed.
+You'll need to install [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node).  We recommend installing [Node.js and npm via a "node version manager"](https://docs.npmjs.com/getting-started/installing-node) to avoid issues with installing packages gloablly. If you choose to install it manually, you might experience the [`EACCES` error when installing packages globally](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-1-change-the-permission-to-npm-s-default-directory).
 
-The Purescript compiler (`purs`) can be installed with npm:
+Install the Purescript compiler (`purs`) with npm:
 
     npm install -g purescript
 
-(It can also be installed from [Hackage](http://hackage.haskell.org/package/purescript), or by downloading the latest [binary bundle](https://github.com/purescript/purescript/releases) for your OS. If you do so, make sure the `purs` executable is on your `$PATH`.)
+Try running the PureScript compiler on the command line to verify that the PureScript compiler executables are available on your path:
```

Comment:
This sentence uses "path", but the following one uses `$PATH`. I think we should just pick one, so that it's clear we're referring to the same thing in each case. I'd recommend `$PATH`, as it ought to be clearer we're using the term in a specific technical sense that way.